spec/util/test_env.lua

```diff
@@ -224,11 +225,13 @@ function test_env.set_args()
          test_env.TRAVIS = true
       elseif argument == "appveyor" then
          test_env.APPVEYOR = true
-         test_env.OPENSSL_DIRS = "OPENSSL_LIBDIR=C:\\OpenSSL-Win32\\lib OPENSSL_INCDIR=C:\\OpenSSL-Win32\\include"
-      elseif argument:find("^os=") then
+			test_env.OPENSSL_INCDIR = "C:\\OpenSSL-Win32\\include"
+		   test_env.OPENSSL_LIBDIR = "C:\\OpenSSL-Win32\\lib"
+		elseif argument:find("^os=") then
          test_env.TEST_TARGET_OS = argument:match("^os=(.*)$")
       elseif argument == "mingw" then
          test_env.MINGW = true
+			test_env.OPENSSL_LIBDIR = "C:\\OpenSSL-Win32\\lib\\MinGW"
```

Comment:
indentation is off — hard tabs sneaked in, should be spaces

Reply:
Sorry about that, I changed my text editor and I don't have it fully configured yet. I'm on it right now.